### PR TITLE
Add support for negative numbers

### PR DESCRIPTION
**Pull request by GorlikItsMe**
Why you dont have support for negative numbers?
I add it can you merge it?

### CODE REVIEW

**Comment by GorlikItsMe** [comment]
Bump?

**Comment by GorlikItsMe** [comment]
Bump

File: slownie.py [hunk 1]
```diff
@@ -91,7 +91,9 @@ def slownie999(i):
 def slownie(i, jeden=False, unit=None):
     i = int(i)
     if i < 0:
-        raise ValueError('i < 0')
+        i = i * -1
+        return u' '.join(["minus", slownie(i, jeden, unit)])
+        # raise ValueError('i < 0')
```
Comment (jwilk) [inline slownie.py:96]: No commented out code, please. That's just distraction for people reading the code, and even greater distraction for people reading the PR diff.

File: test/test.py [hunk 2]
```diff
@@ -86,10 +86,17 @@ def test_normal(self):
...
+        #def f_min():
+        #    slownie.slownie(-1)
         self.assertRaises(ValueError, f_max)
-        self.assertRaises(ValueError, f_min)
+        #self.assertRaises(ValueError, f_min)
+
+    def test_minus(self):
+        expected0 = "minus pięć"
+        if str is bytes:
+            expected0 = expected0.decode('UTF-8')
```
Comment (jwilk) [inline test/test.py:98]:
```suggestion
        expected0 = u'minus pięć'
```

File: slownie.py [hunk 3]
```diff
@@ -91,7 +91,9 @@ def slownie999(i):
 def slownie(i, jeden=False, unit=None):
     i = int(i)
     if i < 0:
-        raise ValueError('i < 0')
+        i = i * -1
+        return u' '.join(["minus", slownie(i, jeden, unit)])
```
Comment (jwilk) [inline slownie.py:95]:
```suggestion
        return u'minus ' + slownie(-i, jeden, unit)])
```

Comment (jwilk) [inline slownie.py:95]: I appreciate the minimalism of the recursive approach, but it leads to suboptimal error messages:
```
>>> slownie.slownie(-10 ** 100)
Traceback (most recent call last):
  File ".../slownie.py", line 115, in slownie
    words = [t, inflect(j, SLOWNIE_1000XX[m])] + words
IndexError: list index out of range

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File ".../slownie.py", line 95, in slownie
    return u' '.join(["minus", slownie(i, jeden, unit)])
  File ".../slownie.py", line 117, in slownie
    raise ValueError('i >= 10 ** {n}'.format(n=3 * len(SLOWNIE_1000XX)))
ValueError: i >= 10 ** 66
```

**Review by jwilk — COMMENTED** [review]
I've committed a different fix in c1dd7c64677468ee6fb619d312baba2561f22e6c.

This review is for education purposes only.

**Pull request closed without merging**